Wk2 438 isa sandbox warning (#24)
wealthkernel/api-documentation-guides: docs/isa-subscriptions/Basics.md

# ISA Subscription Basics

<!-- theme: info -->

> > The ISA Subscriptions API is currently in beta and only available in our sandbox environment.

Reviewer: ```suggestion
> This API is currently in **Beta** and only available in our sandbox environment.
```
